subsys/bindesc/Kconfig

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2023 Yonatan Schachter
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig BINDESC
```

Comment:
Is this really a subsystem?

Reply:
It's really more of a build system feature, so I though the most proper place would be with the subsystems. But if you think a different location would be more suitable, I'll move it.
Also note that in the future we might add code for reading those descriptors from the device itself, if that makes a difference.